gui/main.js

```diff
@@ -340,7 +342,12 @@ app.on('ready', () => {
       updaterWindow.hide()
       showWindowStartApp()
     })
-    updaterWindow.checkForUpdates()
+    if (process.env.COZY_DESKTOP_PROPERTY_BASED_TESTING) {
```

Comment:
I did not undestand why this condition has been added.

Reply:
I didn't find why, but during the tests, the client was sometimes blocked on the update check.